cmds/core/kexec/kexec_linux.go

```diff
@@ -71,6 +73,7 @@ func registerFlags() *options {
 	flag.StringVar(&o.initramfs, "initramfs", "", "Use file as the kernel's initial ramdisk")
 	flag.BoolVarP(&o.load, "load", "l", false, "Load the new kernel into the current kernel")
 	flag.BoolVarP(&o.loadSyscall, "loadsyscall", "L", false, "Use the kexec_load syscall (not kexec_file_load)")
+	flag.StringVarP(&o.loadedInfoFile, "imageInfoFile", "I", "", "Load linux image info from info file given")
```

Comment:
make variable name == flag name? o.imageInfoFile?

Reply:
Make the description: "Load Linux image as described by JSON info file given"

Reply:
Done.

desc slightly reworded: Load Linux Image from JSON info file given.